server/test/controllers/admin/AdminProgramTranslationsControllerTest.java

```diff
@@ -264,7 +268,7 @@ public void update_outOfSyncFormData_redirectsToFreshData() throws Exception {
             .bodyForm(
                 ImmutableMap.<String, String>builder()
                     .put("displayName", "updated spanish program display name")
-                    .put("displayDescription", "updated spanish program description")
```

Comment:
remove `displayDescription` because it's no longer required for submitting translations